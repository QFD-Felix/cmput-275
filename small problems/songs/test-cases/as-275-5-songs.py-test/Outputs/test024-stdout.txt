17 3 11 40 27 3 0 0 11 1 26 26 22 8 10 27 2 3 11 7 2 31 23 2 4 5 6 17 7 66 28 26 1 7 17 6 3 26 23 26 8 5 27 11 38 31 1 19 2 0 5 9 21 0 56 15 0 18 16 0 2 3 18 18 3 3 51 17 4 10 1 14 6 20 42 2 29 9 11 13 3 13 8 4 13 6 2 9 11 50 25 21 26 13 9 13 11 1 3 1 22 3 0 26 5 29 21 41 5 2 10 14 66 8 0 66 7 2 15 11 32 32 2 20 2 12 18 5 0 12 5 0 17 5 16 1 15 86 2 35 7 4 0 15 24 18 0 10 58 6 11 5 2 7 5 10 13 40 4 5 7 5 15 8 16 14 21 6 7 20 0 1 10 44 18 1 14 6 42 0 6 28 18 1 11 0 2 6 3 5 14 7 20 14 16 1 5 43 7 0 22 1 1 8 4 28 22 22 17 6 8 10 1 1 5 12 0 22 16 7 4 11 21 33 12 16 4 34 16 2 18 6 3 18 11 27 13 1 28 24 2 13 13 4 27 5 0 13 4 0 3 7 12 0 3 13 14 2 0 0 0 8 3 11 0 0 1 0 0 4 2 5 0 2 0 0 0 0
