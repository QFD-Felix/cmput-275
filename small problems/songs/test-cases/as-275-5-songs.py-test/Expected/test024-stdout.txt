18 3 3 11 40 27 3 0 0 11 1 26 26 22 8 10 27 2 3 11 7 2 31 23 2 4 5 6 17 7 68 28 26 1 7 17 6 3 26 23 26 8 5 27 11 38 31 1 19 2 0 5 9 21 0 58 15 0 18 16 0 2 3 18 18 3 3 54 17 4 10 1 14 6 22 45 2 31 9 10 15 3 15 10 4 15 8 4 11 1 2 11 53 26 21 27 13 9 13 11 1 3 1 23 3 0 27 5 30 22 44 5 2 5 15 71 9 0 71 5 7 2 15 11 34 34 2 22 2 12 20 5 0 14 5 0 19 7 18 1 3 1 4 90 13 26 7 4 0 15 26 18 0 10 62 6 11 5 2 7 5 12 15 44 4 5 6 7 17 10 18 16 10 2 21 6 7 20 0 1 10 25 18 1 14 6 44 0 6 30 20 1 11 0 2 6 3 5 16 6 22 16 18 1 7 45 2 20 7 0 22 1 1 8 4 28 22 22 17 6 8 10 1 1 5 12 0 22 16 7 4 11 21 34 12 16 4 35 16 2 18 6 3 19 11 28 14 1 29 25 2 14 7 4 28 6 0 14 5 0 6 3 7 12 0 3 13 14 2 0 0 0 8 3 11 0 0 1 0 0 4 2 5 0 2 0 0 0 0
